src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenConstLocal.vb

```diff
@@ -181,7 +181,7 @@ End Module
 
         End Sub
 
-        <Fact()>
```

Comment:
Rather then skipping this test, shouldn't we use platform agnostic raw display?

Reply:
@tmat this ended up being an issue where the test output itself was not using G15. Using G15 ensures that everything works as expected, and I've re-enabled the tests.